### PR TITLE
feat(dbless): adjust docker-compose to capture how to run Kong dbless

### PR DESCRIPTION
**Pull request by hutchic**
- requires docker-compose >= 1.28
- need to update docs
- there's no test that dbless mode works

### CODE REVIEW

**Review by gAmUssA — APPROVED** [review]
LGTM

**Comment by gAmUssA** [comment]
@hutchic any plans to merge this?

**Comment by hutchic** [comment]
@gAmUssA getting there -- trying to get the tests green

**Review by kikito — APPROVED** [review]

File: tests/01-image.test.sh [hunk 1]
```diff
@@ -28,8 +28,33 @@ function run_test {
...
+  curl -I localhost:8001 | grep -E '(openresty|kong)'
+  if [ $? -eq 0 ]; then
+    tsuccess
+  else
+    tfailure
+  fi
+  
+  docker-compose kill
+  docker-compose rm -f
+  sleep 5
```
Comment (kikito) [inline tests/01-image.test.sh:50]: I pressume there isn't something we can actively wait for instead of this.

Comment (hutchic) [reply tests/01-image.test.sh:50]: whoops I missed this comment.

Not easily no (or at least that I know of).

Comment (gszr) [reply tests/01-image.test.sh:50]: Maybe along the lines of https://github.com/Kong/kong-build-tools/blob/master/test/util.sh#L45?

**Pull request merged**